events/teamLeave.js

```diff
@@ -0,0 +1,22 @@
+function teamLeave(context, payload) {
+    // Init variables
+    const playerId = context.id();
+    const gameState = context.getGameState();
+    const player = gameState.players[playerId]
+    const { team } = gameState.players[playerId];
+    const { name } = gameState.teams[team];
+
+    // Update player, -1 for no team
+    player.team = -1;
+
+    // Update game state
+    gameState.players[playerId] = player;
+    context.updateGameState(gameState);
+
+    // Broadcast event
+    context.broadcast('team:left', { playerId, name });
```

Comment:
This wasn't inside the issue (your code has no issues), but I think it might be better to send the team index, instead of the name. Just change the broadcast to `context.broadcastToGame({ playerId, team  })` (notice `broadcastToGame` as well).

Reply:
Alright, seems appropiate. I'll push the changes shortly.